src/commands/deployment/README.md

```diff
@@ -1,6 +1,6 @@
 # Deployment command
 
-Marks a CI job as a deployment.
+Contains some tools related to CD visibility.
```

Comment:
```suggestion
Contains helper commands related to CD Visibility.
```

We can have CD Visibility pointing to https://docs.datadoghq.com/continuous_delivery/.